pro_tes/utils/db_utils.py

```diff
@@ -0,0 +1,114 @@
+"""Utility functions for MongoDB document insertion, updates and retrieval."""
```

Comment:
These will probably have to be adapter for TES vs WES. At least with respect to names, e.g., in line 86, but probably in multiple places